packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/ActionItem.tsx

```diff
@@ -99,58 +113,60 @@ interface IconComponentProps extends CommonProps {
     iconComponent?: never;
 }
 
-type Props = CustomIconComponentProps | IconComponentProps;
+type ActionItemProps = CustomIconComponentProps | IconComponentProps;
 
 // Reason to use forwardRef: We want the user to be able to close Notifications dropdown by clicking somewhere else.
 // In order to achieve that behavior, we need to pass reference to ActionItem
-export const ActionItem = React.forwardRef((props: Props, ref: React.Ref<HTMLDivElement>) => {
-    const theme = useTheme();
-    const iconComponent = props.icon ? (
-        <Icon
-            color={props.isActive ? theme.TYPE_DARK_GREY : theme.TYPE_LIGHT_GREY}
-            size={24}
-            icon={props.icon}
-        />
-    ) : (
-        props.iconComponent
-    );
-
-    if (props.isMobileLayout) {
-        return (
-            <MobileWrapper {...props}>
-                <MobileIconWrapper isActive={props.isActive}>
-                    {iconComponent}
-                    {props.indicator === 'alert' && (
-                        <AlertDotWrapper>
-                            <AlertDot />
-                        </AlertDotWrapper>
-                    )}
-                    {props.indicator === 'check' && (
-                        <Indicator>
-                            <Icon icon="CHECK" size={10} color={theme.TYPE_GREEN} />
-                        </Indicator>
-                    )}
-                </MobileIconWrapper>
-                <Label>{props.label}</Label>
-            </MobileWrapper>
+export const ActionItem = React.forwardRef(
+    (props: ActionItemProps, ref: React.Ref<HTMLDivElement>) => {
+        const theme = useTheme();
+
+        const iconComponent = props.icon ? (
+            <Icon
+                color={props.isActive ? theme.TYPE_DARK_GREY : theme.TYPE_LIGHT_GREY}
+                size={24}
+                icon={props.icon}
+            />
+        ) : (
+            props.iconComponent
         );
-    }
 
-    return (
-        <HoverAnimation isHoverable={!props.isOpen}>
-            <Wrapper isActive={props.isActive} isOpen={props.isOpen} ref={ref} {...props}>
+        const Content = (
+            <>
                 {iconComponent}
                 {props.indicator === 'alert' && (
                     <AlertDotWrapper>
                         <AlertDot />
                     </AlertDotWrapper>
                 )}
+                {props.indicator === 'loading' && (
+                    <Indicator>
+                        <FluidSpinner size={6} />
+                    </Indicator>
+                )}
                 {props.indicator === 'check' && (
                     <Indicator>
                         <Icon icon="CHECK" size={10} color={theme.TYPE_GREEN} />
                     </Indicator>
                 )}
-            </Wrapper>
-        </HoverAnimation>
-    );
-});
+            </>
+        );
+
+        if (props.isMobileLayout) {
```

Comment:
Does not work correctly on mobile. Not sure we need it there?
https://user-images.githubusercontent.com/33235762/175229779-bdd36bec-7575-439b-bf20-a33dcaac447e.mov

Reply:
I don't know, why it's also not just a link to settings on mobile 🤷‍♂️